testcases/kernel/syscalls/syslog/syslog12.c

```diff
@@ -79,6 +79,7 @@
 #include <unistd.h>
 #include <signal.h>
 #include <linux/unistd.h>
+#include <sys/syscall.h>
 #include "test.h"
 
 #define EXP_RET_VAL	-1
```

Comment:
Can you please also split the patch so that changes in build system and additions of random system headers into a testcases are separated?